addons/sys_intercom/script_acre_intercom_defines.hpp

```diff
@@ -5,15 +5,11 @@
 #define INTERCOM_RX_AND_TX      3
 #define INTERCOM_DEFAULT_VOLUME 1
 
-#define STATION_INTERCOM_CONFIGURATION_INDEX  0
-#define STATION_INTERCOM_UNIT_INDEX           1
-#define STATION_RACKS_CONFIGURATION_INDEX     2
-
-#define INTERCOM_STATIONSTATUS_HASINTERCOMACCESS  0
-#define INTERCOM_STATIONSTATUS_CONNECTION         1
-#define INTERCOM_STATIONSTATUS_VOLUME             2
-#define INTERCOM_STATIONSTATUS_LIMITED            3
-#define INTERCOM_STATIONSTATUS_TURNEDOUTALLOWED   4
-#define INTERCOM_STATIONSTATUS_FORCEDCONNECTION   5
-#define INTERCOM_STATIONSTATUS_VOICEACTIVATION    6
-#define INTERCOM_STATIONSTATUS_MASTERSTATION      7
+#define INTERCOM_STATIONSTATUS_HASINTERCOMACCESS  "hasAccess"
```

Comment:
Should we keep those defines?

Reply:
I am undecided. Keeping them allows changing them easily across a bunch of files. I have to check the perforomance impact of direct array access (current implementation) use of cba hash. The later is more flexible